.yamato/_triggers.yml

```diff
@@ -0,0 +1,43 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# Run all relevant tasks when a pull request targets the develop / main or a sample branch
+pull_request_trigger:
```

Comment:
Makes sense to add specific yml files for the triggers and the tests.

However, inside the previous build.yml file, the old pull request trigger job remains. I'm fine with removing that one out of the build.yml file and keeping this here, but let's also move over the standard pass that was recently added as a dependency to our pull request triggers.

On that note, project-standards.yml seems to have been removed. Let's revisit that merge and get that file back, so that the job would fire, since the job def file isn't actually compiling on Yamato (both build.yml & project-standards.yml). See the branch here: https://unity-ci.cds.internal.unity3d.com/project/1573/branch/sample%2FDGS?nav=jobDefinitions.

Last note about the project tests, could we leverage UPM-CI tools here, since I'm seeing that there's an individual job to build a project for every supported platform? You can notice that the [dependency graph](https://unity-ci.cds.internal.unity3d.com/job/35312372/dependency-graph) on Boss Room's pull request trigger job has one packing job, in which the 3 project tests refer to that artifact.

I'm thinking, every project is packed, and we use the booleans (run_editor_tests & run_playmode_tests) inside the projects to determine whether a project test is run, like you've done here. So we would shrink by a lot the number of jobs run.

Reply:
Disclaimer: these changes were made by LP, months ago, so I don't have the full context of "why" they were made.

> On that note, project-standards.yml seems to have been removed. Let's revisit that merge and get that file back, so that the job would fire

I'll open a new branch to start from a point before the merge (reverting it doesn't seem enough, this PR still tries to delete the Anticipation Sample) and open a new PR. If the issues persist there, we'll fix them.

Reply:
> On that note, project-standards.yml seems to have been removed.

project-standards is not present in `main` either. It's only in `develop`.
It seems that `develop` hasn't been merged into `main` yet.